src/frameworks/docs/nuxt.md

```diff
@@ -0,0 +1,64 @@
+# Integrate Nuxt
+
+Using the Firebase CLI, you can deploy your Nuxt apps to Firebase and
+serve them with Firebase Hosting. The CLI respects your Nuxt settings and
+translates them to Firebase settings with zero or minimal extra configuration on
+your part. If your app includes dynamic server-side logic, the CLI deploys that
+logic to Cloud Functions for Firebase.
+
+Note: Framework-aware Hosting is an early public preview. This means
+that the functionality might change in backward-incompatible ways. A preview
+release is not subject to any SLA or deprecation policy and may receive limited
+or no support.
+
+## Before you begin
+
+Before you get started deploying your app to Firebase,
+review the following requirements and options:
+
+- Firebase CLI version 12.1.0 or later. Make sure to
+  [install the CLI](https://github.com/firebase/firebase-tools#installation)
```

Comment:
In a similar PR from Austin I suggested this absolute link for a fix:

https://firebase.google.com/docs/cli#install_the_firebase_cli

But if there's a reason to keep everything in GH, or if only npm works for the frameworks CLI, we can use this (it needs to be fixed in a few other places I'm afraid).